code/Pawn/Worm.cs

```diff
@@ -26,20 +27,13 @@ public override void Respawn()
 			base.Respawn();
 		}
 
-		public void EquipWeapon( Weapon weapon )
-		{
-			// Enable new weapon.
-			EquippedWeapon = weapon;
-			EquippedWeapon?.OnCarryStart( this );
-			EquippedWeapon?.SetWeaponEnabled( true );
-		}
-
-		public void UnequipWeapon()
+		public override void CreateHull()
 		{
-			// Disable old weapon.
-			EquippedWeapon?.OnCarryStop();
-
-			EquippedWeapon = null;
+			CollisionGroup = CollisionGroup.Player;
+			AddCollisionLayer( CollisionLayer.Player );
+			SetupPhysicsFromAABB( PhysicsMotionType.Keyframed, new Vector3( -16, -16, 0 ), new Vector3( 16, 16, 32 ) );
```

Comment:
Might be my lack of experience shining through, but what are these Vector3s? Might be better if they are defined and passed in with their name instead, for future readability. Up to you.

Reply:
The method definition for SetupPhysicsFromAABB kinda covers this for us.
![image](https://user-images.githubusercontent.com/78914427/144313489-3c126449-2c33-434c-817d-e3568095773d.png)